packages/dd-trace/src/appsec/iast/taint-tracking/plugin.js

```diff
@@ -65,6 +65,18 @@ class TaintTrackingPlugin extends SourceIastPlugin {
       }
     )
 
+    this.addSub(
+      { channelName: 'apm:graphql:resolve:start', tag: HTTP_REQUEST_BODY },
+      (data) => {
+        const iastContext = getIastContext(storage.getStore())
+        const source = data.context?.source
+        const ranges = source && getRanges(iastContext, source)
+        if (ranges?.length) {
+          this._taintTrackingHandler(ranges[0].iinfo.type, data.args, null, iastContext)
```

Comment:
Why not `taintObject(iastContext, data.args, ranges[0].iinfo.type)` since `property` will always be `null`?

Reply:
tbh, I didn't realize that I could call directly `taintObject`. But, I think that it is ok if we maintain as it is and call to the same method in almost all `addSub` in the file.